### PR TITLE
Refine continuation get input_str

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

Remove duplicate code in continuation get input_str.
Avoid infinite loop in all-in-one due to test_length not in list.

### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -841,13 +810,6 @@ def run_bigdl_fp16_gpu(repo_id,
             in_len = int(in_out_len[0])
             out_len = int(in_out_len[1])
             # As different tokenizer has different encodings,
-            # in_len.txt maybe shorter than we need,
-            # use much longer context to make sure input length
-            test_length = min(in_len*2, 8192)
-            while test_length not in [32, 256, 1024, 2048, 8192]:
-                test_length = test_length * 2
-            input_str = open(f"prompt/continuation/{test_length}.txt", 'r').read()
```
Comment (plusbang) [inline python/llm/dev/benchmark/all-in-one/run.py:849]: There seems lack `get_continuation_input_str`?

**Review by plusbang — COMMENTED** [review]
Other LGTM

File: python/llm/dev/benchmark/all-in-one/run.py (hunk 1 above)
Comment (qiyuangong) [reply python/llm/dev/benchmark/all-in-one/run.py:849]: Done

**Review by plusbang — APPROVED** [review]
LGTM

**Pull request merged**